libpromises/feature.c

```diff
@@ -27,6 +27,7 @@ static const char* features[] = {
 #endif
     "def_json_preparse",
     "host_specific_data_load",
+    "copyfrom_restrict_keys",
```

Comment:
We should probably remove this, we introduced it to make it easy to have policies specific to our patched agents.

Reply:
I like it, actually. @nickanderson will have the deciding word, though. :)